test/testother.cpp

```diff
@@ -10467,7 +10489,7 @@ class TestOther : public TestFixture {
         check("void foo(char *c) {\n"
               "    if (*c == '+' && (operand || !isalnum(*c))) {}\n"
               "}");
-        ASSERT_EQUALS("", errout.str());
+        ASSERT_EQUALS("[test.cpp:1]: (style) Parameter 'c' can be declared as pointer to const\n", errout.str());
```

Comment:
I wonder if it would be better to adjust the code instead. They are testing unrelated things. Same for the change below.

Reply:
I'd rather not adjust the second example, since there already is a `const` version of the same test.